aws/logs_monitoring/tools/publish_layers.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/bash
+
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2019 Datadog, Inc.
+
+# Publish the datadog python lambda layer across regions, using the AWS CLI
```

Comment:
Looks like this was copypasted from the Python layer, should be updated for the Forwarder

Reply:
This comment should still be updated